astro/src/content/docs/get-started/core-concepts/applications.mdx

```diff
@@ -385,11 +388,11 @@ There are two types of self service registration, basic and advanced.
   <APIField name="Registration fields" optional>
     The optional fields to be displayed on the registration form.
 
-        <InlineField>Field</InlineField> [read-only]#Read-only#::
+    * <InlineField>Field</InlineField> <ReadOnly />
```

Comment:
Looks like this was actually a mistake in the migration. These are supposed to be API fields and not InlineFields
before:
<img width="1053" alt="Screenshot 2023-12-04 at 4 38 35 PM" src="https://github.com/FusionAuth/fusionauth-site/assets/11809481/c11722b4-3806-453d-bd11-6277dda7a2ab">

now:
<img width="883" alt="Screenshot 2023-12-04 at 4 39 54 PM" src="https://github.com/FusionAuth/fusionauth-site/assets/11809481/d1446bc6-d472-4b4c-ae69-18504953e534">

Reply:
Oh, great! fix that right up.